### ISSUE TITLE
Safe Area for iPhone X

### ISSUE THREAD

**Issue by rfrailemartinez** [jdehlin/Xamarin-Sidebar#86]
Menu Controller works perfectly with constraints relative to Safe Area
Content Controller doesn't works. The same controller insert as root windows works well, but if you put it on sidebarController the constrains doesn't works as expected.

**Comment by MobiTusharJ** [comment]
Yes, I am also facing same issue. ContentAreaController is not at all handling Safe Area with the latest release 2.1.0 too. Does anybody found solution, please let me know.
